RELEASE_NOTES.md

```diff
@@ -11,14 +11,24 @@ This is built on the Jupyter Notebook v6.4.12 and IPython 8.5.0 (more notes will
 -   PTV-1909 - Narrative sign out does sign out
 -   UIP-41 - Update Jupyter Notebook to version 6.5.6
 -   UIP-41 - Fix Dropzone dependency issues, address problems with unit tests
+-   UIP-41 - Removed unused Python dependencies
 
 ### Dependency Changes
 
+- Updated to include all Python dependencies in the image built in this repo, and removed those that are not directly referenced in KBase code.
+- Retained scientific computing dependencies that may be useful to the community (SciPy, numpy, pandas, etc.)
+- Removed directly required dependencies that are really just used by the Jupyter Notebook.
+- Separated requirements into image dependencies (i.e. packages that are generally useful to have on the Narrative Docker image, ported from the narrative-base-image repo), application requirements and dev requirements.
+- Image dependencies are all found in the `python_dependencies` directory, and split into 2 files for convenience - base-pip-requirements and kbase-pip-requirements.
+- base-pip-requirements has "base" things - certifi, cryptography, pyopenssl, etc. These are modules that are generally useful as part of the image.
+- kbase-pip-requirements has scientific programming packages that may be of use to the KBase community
+- the requirements in `src/requirements.txt` are needed for the Narrative to run.
+- the requirements in `src/requirements-dev.txt` are needed for Narrative development and testing.
```

Comment:
I wonder if dependabot will still be able to find these requirements files. I have a feeling it is set to look in the `src` directory at present and we want it to check both the `src` and `python_dependencies` dirs.

Reply:
I'd be tempted just to put them all in a single `requirements` file, maybe with sections to show the type/purpose of the module being installed.

Reply:
Yeah, I'm tempted that way, too. I'm not sure we really gain all that much by separating them, other than it was a big list to split up.

Reply:
We can configure dependabot to look for them in different places as well. From reading the docs, just add another stanza with `package-ecosystem: pip` and a different directory.

I'd like to modify the config there, anyway, to make it do other things I read about - like lump together language updates into a single PR (like one for Python, one for JS), and ignore a few dependencies we have to freeze for now (like notebook).